umockdev-based tests appear not to play nicely with the sandbox,
often they work but on some systems they persistently fail.

--- a/meson_options.txt
+++ b/meson_options.txt
@@ -457,6 +457,11 @@
   value: true,
   description: 'enable tests',
 )
+option('test-full',
+  type: 'boolean',
+  value: false,
+  description: 'enable tests known to be fragile when called from Gentoo ebuilds'
+)
 option('curl',
   type: 'feature',
   description: 'libcurl support',
--- a/plugins/thunderbolt/meson.build
+++ b/plugins/thunderbolt/meson.build
@@ -19,7 +19,7 @@
 plugin_builtins += plugin_builtin_thunderbolt
 
 # we use functions from 2.52 in the tests
-if get_option('tests') and run_sanitize_unsafe_tests and umockdev.found() and gio.version().version_compare('>= 2.52')
+if get_option('tests') and get_option('test-full') and run_sanitize_unsafe_tests and umockdev.found() and gio.version().version_compare('>= 2.52')
   env = environment()
   env.set('G_TEST_SRCDIR', meson.current_source_dir())
   env.set('G_TEST_BUILDDIR', meson.current_build_dir())
